mail_forwarding/models/res_user.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    forwarding_partner_id = fields.Many2one(
+        related="partner_id.forwarding_partner_id", readonly=False
+    )
+
+    def __init__(self, pool, cr):
+        super().__init__(pool, cr)
+        self.SELF_WRITEABLE_FIELDS = list(self.SELF_WRITEABLE_FIELDS)
+        self.SELF_WRITEABLE_FIELDS.append("forwarding_partner_id")
+        self.SELF_READABLE_FIELDS = list(self.SELF_READABLE_FIELDS)
+        self.SELF_READABLE_FIELDS.append("forwarding_partner_id")
```

Comment:
I suppose you can remove this file since res.users inherits res.partner.

Reply:
I tried it, but still be necessary having it.